.editorconfig

```diff
@@ -14,7 +14,3 @@ trim_trailing_whitespace = true
 
 [*.{yml,yaml}]
 indent_size = 2
-
-[*.{[Bb][Aa][Tt],[Cc][Mm][Dd]}]
-# DOS/Win *requires* BAT/CMD files to have CRLF newlines
```

Comment:
This is still required though, no? Batch scripts won't work on LF. 
(I'm aware this setting doesn't affect git checkouts).

Reply:
We have set CRLF as the default above, so all text files should be CRLF. As far as I understand 🙂 If we would have set LF as the default, then we would need this block.